train_redpajama.py

```diff
@@ -98,18 +87,28 @@ def main(
     )
     train_dataloader, val_dataloader = fabric.setup_dataloaders(train_dataloader, val_dataloader)
 
+    with fabric.device:
+        torch.set_default_tensor_type(torch.HalfTensor)
```

Comment:
maybe change this to [`torch.set_default_dtype`](https://pytorch.org/docs/stable/generated/torch.set_default_dtype.html#torch.set_default_dtype) as this method will be deprecated (https://github.com/pytorch/pytorch/pull/99521)

Reply:
Yep